logic/src/commonMain/kotlin/com/wire/kalium/logic/util/BooleanExtension.kt

```diff
@@ -0,0 +1,3 @@
+package com.wire.kalium.logic.util
+
+fun Boolean.toInt() = if (this) 1 else 0
```

Comment:
```suggestion
internal fun Boolean.toInt() = if (this) 1 else 0
```

Let's avoid leaking this to consumers of the SDK.